packages/components/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './components';
 export * from './hooks';
 export * from './theme';
-export { keyframes } from '@chakra-ui/react';
-export { Fade, ScaleFade, Slide, SlideFade, Collapse } from '@chakra-ui/react';
+export { keyframes, Fade, ScaleFade, Slide, SlideFade, Collapse, useMergeRefs } from '@chakra-ui/react';
+export { Global } from '@emotion/react';
```

Comment:
why do we need to re-export this?

Reply:
I'm using the `Global` in order to create a `Fonts` component where I set the different font-faces required without requiring to create a `*.css` file